doc/source/contributing.rst

```diff
@@ -349,18 +357,18 @@ Furthermore, it is recommended to have all `optional dependencies
 <http://pandas.pydata.org/pandas-docs/dev/install.html#optional-dependencies>`_
 installed. This is not strictly necessary, but be aware that you will see some error
 messages. Because all the code in the documentation is executed during the doc
-build, the examples using this optional dependencies will generate errors.
+build, the examples using the optional dependencies will generate errors.
 Run ``pd.show_versions()`` to get an overview of the installed version of all
 dependencies.
 
 .. warning::
 
-   Sphinx version >= 1.2.2 or the older 1.1.3 is required.
+   Sphinx version >= 1.2.2, or older than 1.1.3, is required.
```

Comment:
There is already https://github.com/pydata/pandas/issues/10316, which is about broken doc build with 1.3 which should be solved first (and actually this sphinx specification could also include `< 1.3`)